packages/framework-core/src/services/graphql/graphql-type-informer.ts

```diff
@@ -107,4 +141,13 @@ export class GraphQLTypeInformer {
     }
     return inputFields
   }
+
+  private generateOperationEnumValuesFor(operationsEnum: AnyClass): GraphQLEnumValueConfigMap {
+    const enumValuesConfig: GraphQLEnumValueConfigMap = {}
+    for (const opSymbol in operationsEnum) {
+      const opName = (operationsEnum as any)[opSymbol]
+      enumValuesConfig[opName] = { value: opSymbol }
+    }
+    return enumValuesConfig
+  }
```

Comment:
Is this function still needed? It was used for the old filteres

Reply:
It's still used when there is an Enum